syntax = "proto3";

enum UserStatus {
  USER_STATUS_INVALID = 0;    // 非法状态
  USER_STATUS_ENROLL = 1;     // 待注册
  USER_STATUS_LOGIN = 2;      // 登录
  USER_STATUS_LOGOUT = 3;     // 登出
  USER_STATUS_PASSWD = 4;     // 修改密码：
}

message AccountInfo {
  string user = 1;                  // 用户名
  string phone = 2;                 // 用户名
  string password = 3;              // 用户密码
  string tmp_password = 4;          // 新密码
  string token = 5;                 // 登录签名
  string tmp_token = 6;             // 登录签名
  string code = 7;                  // 验证码
  UserStatus status = 8;            // 用户状态
  int32 try_passwd_count = 9;       // 试密码次数
  int64 try_passwd_block_time = 10; // 上次被锁时间
}                                   

